packages/workers/src/scheduler.ts

```diff
@@ -1,19 +1,40 @@
 import cron from 'node-cron';
+import dotenv from 'dotenv';
 import express from 'express';
 
 import runTgAlerts from './scripts/il-alerts';
 import runDiscordAlerts from './scripts/il-alerts-discord';
+import { run as runRedisCacheWarmer } from './scripts/redis-cache-warmer';
+
+dotenv.config();
 
 const PORT = 8080;
 const CRON_EVERY_HOUR = '0 * * * *';
 
-cron.schedule(CRON_EVERY_HOUR, () => {
-    void runTgAlerts();
-});
+let tgAlertsEnabled = process.env.WORKER_TG_ALERTS ?? 'false';
+tgAlertsEnabled = JSON.parse(tgAlertsEnabled);
 
-cron.schedule(CRON_EVERY_HOUR, () => {
-    void runDiscordAlerts();
-});
+if (tgAlertsEnabled) {
+    cron.schedule(CRON_EVERY_HOUR, () => {
+        void runTgAlerts();
+    });
+}
+
+let discordAlertsEnabled = process.env.WORKER_DISCORD_ALERTS ?? 'false';
```

Comment:
@kkennis we will need to enable these vars in the cloudbuild yaml

Reply:
So we want the cloudbuild to set this to true for the v2 deployment and false for the regular deployment, right?

Reply:
i think just the v3 deployment needs to be set cuz v2 is pinned to an old version without this change